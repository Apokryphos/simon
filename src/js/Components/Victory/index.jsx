import React from 'react';

function Victory() {
  return <span className="victory">You won!</span>;
}

export default Victory;
