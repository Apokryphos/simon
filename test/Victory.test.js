import React from 'react';
import { shallow } from 'enzyme';
import Victory from 'Components/Victory';

describe('Victory rendering', () => {
  it('Renders OK', () => {
    const wrapper = shallow(<Victory />);
    expect(wrapper).toMatchSnapshot();
  });
});
