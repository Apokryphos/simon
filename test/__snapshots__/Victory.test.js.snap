// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Victory rendering Renders OK 1`] = `
<span
  className="victory"
>
  You won!
</span>
`;
